Dockerfile

```diff
@@ -48,6 +48,7 @@ COPY malcolmjs /malcolmjs
 # Toolchains and tar files
 RUN bash scripts/GNU-toolchain.sh
 RUN bash scripts/tar-files.sh
+RUN bash scripts/install-ghdl.sh
 
 # For the documentation
 RUN pip3 install matplotlib \ 
```

Comment:
could you also add a new run line that install cocotb? something like `RUN pip3 install cocotb`

Reply:
done